Sources/SwiftInspectorCommands/StaticUsageCommand.swift

```diff
@@ -53,9 +53,6 @@ final class StaticUsageCommand: ParsableCommand {
 
   /// Validates if the arguments of this command are valid
   func validate() throws {
-    guard !statics.isEmpty else {
-      throw InspectorError.emptyArgument(argumentName: "--statics")
-    }
```

Comment:
We don't need this validation since now this is already being done by SwiftArgumentParser https://github.com/apple/swift-argument-parser/blob/main/CHANGELOG.md#changes

I noticed this because we had a test for this use case and the code coverage dropped, since now it's actually impossible to trigger this path since it will be caught by SwiftArgumentParser before getting here.